src/java/org/apache/cassandra/exceptions/ConfigurationException.java

```diff
@@ -35,7 +35,7 @@ public ConfigurationException(String msg)
     public ConfigurationException(String msg, boolean logStackTrace)
```

Comment:
This is now a duplicate of the constructor above? Also, looks like we're in a state where we just...always log now?

Reply:
Not intentional change, I will revert it